mypy/find_sources.py

```diff
@@ -127,6 +127,10 @@ def crawl_up_dir(self, dir: str) -> Tuple[str, str]:
             res = ''
             base_dir = dir or '.'
         else:
+            # Remove -stubs to treat PEP-561 stub-only directories as packages
+            if base.endswith('-stubs'):
+                base = base[:-6]
+
             # Ensure that base is a valid python module name
```

Comment:
Somehow I initially misread this as having a global effect -- but in fact `base` is only used in the check `base.isidentifier()` below, so it's a throwaway variable. How about moving stuff around so the other comment still leads?

```suggestion
            # Ensure that base is a valid python module name
            if base.endswith('-stubs'):
                base = base[:-6]  # PEP-561 stub-only directory
```

Reply:
Was the removal of the empty line after the assignment intentional?

Reply:
Yes.